src/components/Table/GridTable.test.tsx

```diff
@@ -2469,6 +2471,7 @@ describe("GridTable", () => {
             id: "2",
             data: "Tony Stark",
             initSelected: false,
+            inferSelectedState: false,
```

Comment:
I had to update this test b/c w/o `inferSelectedState: false`, the `initSelected: false` recommendation was being overridden.

I believe this test update is fine, unless we want to say something like "setting `initSelected` _also_ means we should consider this row to be `inferSelectedState: false`? 

That's a doable change if we want...wdyt?